bin/cleanup

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Eventually it would be nice that each test takes care of its own cleanup, but in the meantime, it's nice to have 👍

Reply:
Oh, I never meant to ``git add`` this. Ah well, it doesn't hurt.